packages/firestore/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
   if you continue to use `experimentalTabSynchronization`.
 - [feature] You can now query across all collections in your database with a
```

Comment:
To double the impact of your PR, can you move these two items to a new section? They were releases as part of 1.3.0 if I am not mistaken.

Also (@mikelehen) - Should we just use the version numbers from https://firebase.google.com/support/release-notes/js? In which case this would 6.0. We don't seem to have a changelog for the Firestore NPM package.

Reply:
Personally I would be fine with that and it would be a lot easier. But I also don't want to volunteer to go back and fix all the version numbers. 😛